tests/test_algorithm.py

```diff
@@ -4359,13 +4359,11 @@ def test_daily_delisted_equities(self,
             )
 
         # Check expected cash.
-        self.assertEqual(algo.cash, expected_cash)
```

Comment:
Ah that makes sense, I'll leave it up to you as to whether you think the second check is worth it.